projects/cdk/utils/index.ts

```diff
@@ -7,4 +7,5 @@ export * from '@taiga-ui/cdk/utils/format';
 export * from '@taiga-ui/cdk/utils/math';
 export * from '@taiga-ui/cdk/utils/miscellaneous';
 export * from '@taiga-ui/cdk/utils/os';
+export * from '@taiga-ui/cdk/utils/sorting';
```

Comment:
I don't think we need it, maybe in `miscellaneous`?

Reply:
fixed